### ISSUE TITLE
a bit confused

### ISSUE THREAD

**Issue by outbackdingo** [STRRL/cloudflare-tunnel-ingress-controller#77]
i added 

```kubectl -n kubernetes-dashboard \
  create ingress dashboard-via-cf-tunnel \
  --rule="klab.optimcloud.com/*=kubernetes-dashboard:443"\
  --class cloudflare-tunnel
```
which results in

│ 2024-05-13T13:41:15Z INF Updated to new configuration config="{\"ingress\":[{\"hostname\":\"klab.optimcloud.com\", \"path\":\"/\", \"service\":\"http://10.103.59.236:443\"}, {\"service\":\"http_status:404\"}], \"originRequest\":{}, \ │

however k8s says the svc is running

kubernetes-dashboard                   kubernetes-dashboard                        ClusterIP   10.103.59.236    <none>        443/TCP

**Comment by STRRL** [comment]
Hi @outbackdingo, I assume that you have a little confusion about the last rule "http_status:404".

cloudflared accept configurations from cloudflare tunnel dashboard, and it looks like:


```json
{
  "ingress":[
    {"hostname":"klab.optimcloud.com", "path":"/", "service":"http://10.103.59.236:443"}, 
    {"service":"http_status:404"}
  ]
}
```

here is a list of rules called `ingress`, and when request hits the cloudflared, it would try to resolve the request based on the rules, from top to down, one by one, and the last rule is a fallback rules.

So basically, in your situation, it means, cloudflare would redirect any request matched with header "HOST: klab.optimcloud.com" to the target `http://10.103.59.236:443`, and respond 404 for the rest requests which are not matched.

**Comment by STRRL** [comment]
I am going to close this issue, feel free to reopen it if you still have concerns.
